### PR TITLE
Added short description for tasks

### PR DESCRIPTION
**Pull request by merveenoyan**
I wrote very short description to add tasks in where you have pointed @coyotte508

### CODE REVIEW

File: docs/inference/README.md [hunk 1]
```diff
@@ -180,3 +180,7 @@ await hf.textToImage({
 ```console
 HF_ACCESS_TOKEN="your access token" npm run test
 ```
+
+# Finding model appropriate for your task
```
Comment (coyotte508) [inline docs/inference/README.md:184]:
```suggestion
## Finding model appropriate for your task
```

File: docs/inference/README.md [hunk 2]
```diff
@@ -180,3 +180,7 @@ await hf.textToImage({
...
+
+We have an informative documentation project called [Tasks](https://huggingface.co/tasks) to explain how each task works and how to infer with the models, for software developers.
```
Comment (coyotte508) [inline docs/inference/README.md:186]:
```suggestion
If you want to try out different models for each of the APIs, you can look at our [Tasks](https://huggingface.co/tasks) pages to easily find compatible models.
```

I think what the users of the JS libs will want is just to find a model for the task, they don't care about the inner workings, just that it works. As for the inputs, it's probably handled by the Typescript definitions. 

If there are model-specific inputs, then we should maybe write an additional documentation page (if the lib supports arbitrary inputs).

cc @radames or @TimMikeladze  to make sure I don't say anything ridiculous

Comment (radames) [reply docs/inference/README.md:186]: Agree with @coyotte508 , that if the model-specific inputs should be explained in the docs here. But I like @merveenoyan docs explaining each task especially for a users landing in ML world via our JS library.

Comment (coyotte508) [reply docs/inference/README.md:186]: Alright, let me rephrase: 

>  to explain how each task works

The thing is `tasks` is mentioned nowhere before in the README, and here we talk about tasks out of the blue. For example we have a section called `## Supported APIs` above, maybe we should rename it `## Supported Tasks` ?

> and how to infer with the models, for software developers.

The tasks page doesn't do that. The README of the `@huggingface/inference` package explains how to infer, by calling `hf.questionAnswering(...)`.

The advantage of the tasks page is to explain what each tasks does, and to list models for each tasks.

Comment (julien-c) [reply docs/inference/README.md:186]: maybe orthogonal to this, but i do think we should have a model id as default (for users who just want to do a task but know no models)

Sorry if it's off topic here

Comment (julien-c) [reply docs/inference/README.md:186]: hmm no i think it's better if people can dive a bit and introspect stuff and realize it's actually a defined model not a black box API

Unlike OpenAI we want to empower people to actually do some ML not just consume it

**Comment by merveenoyan** [comment]
fixes #51

File: docs/inference/README.md (hunk 2 above)
Comment (coyotte508) [reply docs/inference/README.md:186]:
```suggestion
We have an informative documentation project called [Tasks](https://huggingface.co/tasks) to list available models for each task and explain how each task works in detail.

It also contain demos, example outputs and other resources should you want to dig more into the ML-side of things.
```

Something like this?

I just don't agree with "to infer with the model for a given task for a software developer" because it's this README's job / `@huggingface/infernece`'s job to do that, at least for JS / TS developers. 

The TS type inference will autocomplete their code to provide the correct parameters / call the correct functions.

**Review by coyotte508 — APPROVED** [review]

**Comment by coyotte508** [comment]
Thanks @merveenoyan !
